tasks/release.py

```diff
@@ -40,6 +40,9 @@
 # Regex matching rc version tag format like 7.50.0-rc.1
 RC_VERSION_RE = re.compile(r'\d+[.]\d+[.]\d+-rc\.\d+')
 
+# Regex matching minor release rc version tag (no patch version) like 7.50-rc.1, but not 7.50.1-rc.1
```

Comment:
I think the regex will match `7.50.0-rc.1` and not `7.50.1-rc.1`. But that's not exactly what the comment says, `7.50-rc.1` is not matched by the regex

Reply:
This regex: `\d+[.]\d+-rc\.\d+` would do what the comment says

Reply:
Right, thats for spotting that. The issue was actually in the comment, not the regex.

Reply:
I think that now you have an issue with the other suggestions that Nicolas made

Reply:
I checked the descriptions once more, they should be accurate now